### ISSUE TITLE
Ansible fails on "Lookup control node from file"

### ISSUE THREAD

**Issue by samcat116** [PyratLabs/ansible-role-k3s#107]
### Summary
When setting up a one node cluster as described in the documentation, the playbook fails with:
```
TASK [xanmanning.k3s : Lookup control node from file] ***************************************************
fatal: [node-3]: FAILED! => {
    "changed": false,
    "rc": 256
}

MSG:

no command given
```

### Issue Type

- Bug Report

### Controller Environment and Configuration

```
PLAY [Setup k3s] ****************************************************************************************

TASK [Gathering Facts] **********************************************************************************
ok: [node-3]

TASK [xanmanning.k3s : Check that k3s_state is a supported value] ***************************************
ok: [node-3] => {
    "changed": false
}

MSG:

k3s_state is valid.

TASK [xanmanning.k3s : Check that Ansible v2.10.2 is supported by this role] ****************************
ok: [node-3] => {
    "changed": false
}

MSG:

Ansible v2.10.2 is supported.

TASK [xanmanning.k3s : include_tasks] *******************************************************************

TASK [xanmanning.k3s : include_tasks] *******************************************************************
fatal: [node-3]: FAILED! => {
    "reason": "this task 'ansible.builtin.command' has extra params, which is only allowed in the following modules: ansible.windows.win_shell, raw, include_vars, shell, include_tasks, win_shell, script, import_tasks, group_by, set_fact, import_role, command, include, ansible.windows.win_command, add_host, win_command, include_role, meta\n\nThe error appears to be in '/Users/sam/.ansible/roles/xanmanning.k3s/tasks/validate/environment/local/issue-data.yml': line 6, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Ensure Ansible version is captured\n  ^ here\n"
}

PLAY RECAP **********************************************************************************************
node-3                     : ok=3    changed=0    unreachable=0    failed=1    skipped=1    rescued=0    ignored=0   
```

Ansible Version: 2.10.2
Role version: 2.8.1

### Steps to Reproduce

```  - name: Setup k3s
    hosts: node-3
    vars:
      k3s_become_for_all: true
    roles:
      - xanmanning.k3s

```

### Expected Result

```
K3s successfully deploys
```

### Actual Result

```
TASK [xanmanning.k3s : Lookup control node from file] ***************************************************
fatal: [node-3]: FAILED! => {
    "changed": false,
    "rc": 256
}

MSG:

no command given
```

**Comment by xanmanning** [comment]
What version of Ansible are you running?

**Comment by xanmanning** [comment]
Sorry, I missed your ansible version. See: https://github.com/PyratLabs/ansible-role-k3s/issues/105#issuecomment-817182233

And see: https://github.com/PyratLabs/ansible-role-k3s#requirements

TL;DR - Ansible broke FQCNs for specific releases, I am not going to support 2.10.2.
